template/src/services/networking/index.js

```diff
@@ -0,0 +1,106 @@
+import {isObject} from '../../utilities/utils'
+import instance from './axios'
+
+export const TokenType = {
+  Paymentez: 'Paymentez',
```

Comment:
what **Paymentez** is for?

Reply:
We don't really need it on Base Project.

Reply:
done